### DISCUSSION TITLE
Ist es möglich, den Service Typ zu bearbeiten?

### DISCUSSION THREAD

**Discussion by Fan8099** [PredatH0r/ChanSort#341] [Q&A]
Hi, ich würde gerne den Service Typ eines Kanals (auf meinem Samsung-Fernseher) von SD-TV zu HD-TV ändern. Ist das möglich?

Hier ist warum: Der Kanal strahlt in 4:3 aus, obwohl das Bild für 16:9-Bildschirme gedacht ist. Ich hoffe darauf, dass wenn ich meinen Fernseher denken lasse, dass es ein HD-Kanal ist, dass er das Bild automatisch auf 16:9 streckt, auch wenn es nicht das "empfohlene" Seitenformat des Anbieters ist.
Ich weiß, dass es eine Option gibt, 4:3 auf 16:9 zu strecken, aber diese will ich nicht aktivieren, da ich andere Sender (die ein 4:3-Bild haben) damit nicht strecken will.

Falls es mit ChanSort nicht geht, kann man es selber bearbeiten? Vielleicht mit einem Hex-Editor? Falls es wichtig zu wissen ist: Der Fernseher ist aus der Q-Reihe, also sind die Kanäle in eine ZIP-Datei exportiert worden.

**Comment by PredatH0r** [comment]
Ich beschränke mich mit ChanSort bewusst auf die grundlegenden Funktionen zum Sortieren von Sendern.
Das Ändern von Tuner-Informationen ist viel zu gerätespezifisch und auch für mich unmöglich auszutesten, ohne die entsprechenden Fernseher zu besitzen.

Abgesehen davon wird das Ändern des Servicetypes in der Senderliste vermutlich nichts bringen. Der Wert wird automatisch im DVB Datenstrom mitgeschickt und vermutlich wird der Fernseher es sofort wieder überschreiben, sobald man auf den Kanal schaltet.

Manuell bearbeiten kann man die Daten indem man die .zip Datei entpackt und dann z.B. die "dvbs" mit einem Programm wie "DB Browser for SQLite" oder "SQLite Studio" öffnet. In der "Data"-Ansicht sieht das dann aus eine Excel-Tabelle, in der man die Daten auch ändern kann.
Beim Anschließenden zippen muss man aufpassen, dass da nicht ein extra Verzeichnis innerhalb des zips landet. Es muss wieder genau so aufgebaut sein wie im Original.
